src/util/Util.js

```diff
@@ -53,10 +53,10 @@ class Util {
    * Splits a string into multiple chunks at a designated character that do not exceed a specific length.
    * @param {string} text Content to split
    * @param {SplitOptions} [options] Options controlling the behavior of the split
-   * @returns {string|string[]}
+    * @returns {string[]}
```

Comment:
Looks like a space was added accidentally.
```suggestion
   * @returns {string[]}
```